core/src/main/scala/quasar/plugin/postgres/Config.scala

```diff
@@ -78,6 +78,14 @@ object Config {
     implicit val uriEncodeJson: EncodeJson[URI] =
       EncodeJson.of[String].contramap(_.toString)
 
-    casecodec3(Config.apply, Config.unapply)("connectionUri", "schema", "writeMode")
+    CodecJson(cfg =>
+      ("connectionUri" := cfg.connectionUri) ->:
+      ("schema" := cfg.schema) ->:
+      ("writeMode" :=?  cfg.writeMode) ->?:
+      jEmptyObject, c => for {
+        uri <- c.downField("connectionUri").as[URI]
+        schema <- c.downField("schema").as[Option[String]]
```

Comment:
Does this work as specified in the README when `schema` is not provided? I see the type is as expected `Option[String]` but I would have expected `"schema" :=? cfg.schema` instead of `"schema" := cfg.schema`

Maybe add a test for this?